README.md

```diff
@@ -142,7 +142,7 @@ Pretty-printing the `(ns ..)` form is surprisingly difficult.  The current imple
 
 In the event of an error `clean-ns` will return `error` which is an error message intended for display to the user.
 
-**Warning**: The `clean -ns` op dependes on `tools.analyzer` to determine which vars in a file are actually being used.  This means the code is evaluated and any top-level occurrences of `(launch-missiles)` should be avoided.
+**Warning**: The `clean-ns` op dependes on `tools.analyzer` to determine which vars in a file are actually being used.  This means the code is evaluated and any top-level occurrences of `(launch-missiles)` should be avoided.
```

Comment:
This warning isn't applicably anymore as `clean-ns` no longer uses `tools.analyzer`. It does however depend on the code being loaded now (in order to handle :refer :all clauses)